### ISSUE TITLE
[Dev][Tip]Add dropdown to allow user to choose the tip language

### ISSUE THREAD

**Issue by discombobulateme** [saucelabs/elemental-next#556]
The individual tip page's main feature is allowing users to choose which language they would like to see the example using a dropdown selection. 
This issue should implement it, changing the main content while keeping the title.

See screenshot of the design: 

- [ ] Dropdown Component
- [ ] CSS Styles for Dropdown
- [ ] Markdown Wrapper so contributers won't have to import/export

![Image](https://github.com/saucelabs/elemental-next/assets/7980624/4b94312a-c307-4590-ba3e-3145165ac720)

**Comment by discombobulateme** [comment]
One possibility to fix this is propose on PR #599 which required a big work changing the docs architecture.
Team is currently reviewing it

**Comment by discombobulateme** [comment]
Proposed new architecture to docs: 

```bash
/frontend
	/src
	/docs
  		|- 0-tip
     			|- main.mdx
			|- csharp.md
			|- java.md
			|- javascript.md
			|- python.md
			|- ruby.md
  		|- 1-tip
     			|- main.mdx
			|- csharp.md
			|- java.md
			|- javascript.md
			|- python.md
			|- ruby.md
```


Main.mdx file becomes the source of truth and the only displayable sidebar (which helps with the sidebar issue). This is the template content:

```markdown
---
title: 'Template'
id: 'template'
contentUrl: '/docs/template/template'
sidebar_label: template
text: 'This is a template.'
number: 0
hide_table_of_contents: true
level: #add level 1, 2 or 3
publish_date: 2023-05-26 #add date
last_update:
  date: 2023-05-26
tags:
  - 'template'
category: 'template'
---

# Page Title

## Intro

import DropdownMarkdown from '../../src/components/dropdownMarkdown';

<DropdownMarkdown folderName='template' />

````

And the individual language files contain
---
language: ruby
level: #add level 1, 2 or 3
hide_sidebar: true
publish_date: 2023-05-26 #add date
last_update:
  date: 2023-05-26
---

## A Solution
## Example(s)
## Expected Behavior
## Summary
## About The Author
````

If the code in the language does not exists , there’s a default call to action for contribution
````
Oh no! It seems like we are missing code here.
Help the community
<button className='hello'><a href="https://github.com/saucelabs/elemental-next/blob/main/CONTRIBUTING.md" target="_blank" rel="noopener noreferrer">ADD EXAMPLE</a></button>
````
In this branch, I changed tips 71, 72, and 73 and created the template folder.

**Comment by discombobulateme** [comment]
Working on tips 71-60 :)

**Comment by discombobulateme** [comment]
Closed with PR #599
